runner/src/exercism/test_runner.gleam

```diff
@@ -17,6 +17,11 @@ import gleam_community/ansi
 
 pub fn main() {
   let assert Ok(files) = read_directory("test")
+  let files =
+    list.map(files, charlist.to_string)
+    |> list.filter(fn(file) {
+      string.ends_with(file, "test.gleam") && !string.starts_with(file, ".")
+    })
```

Comment:
Maybe instead of just reading in the test directory, the `.meta/config.json` file could be parsed?

Reply:
Hmm, that would preclude students from uploading any additional test files. However I suppose the likelihood of that happening is extremely small.